app/src/main/java/ch/epfl/sweng/SDP/home/HomeActivity.java

```diff
@@ -110,14 +109,11 @@ public static void disableBackgroundAnimation() {
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        overridePendingTransition(0, 0);
         setContentView(R.layout.activity_home);
         profileWindow = new Dialog(this);
         friendRequestWindow = new Dialog(this);
         friendRequestWindow.setCancelable(false);
```

Comment:
Function might be too big. Also consider adding more comments

Reply:
Yes but again, this code was not made during my sprint, I just changed small issues.. We should do that in a refactoring sprint